hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/retransmit/DeleteOfflineRetransmissionTaskCommand.java

```diff
@@ -0,0 +1,32 @@
+package pl.allegro.tech.hermes.management.domain.retransmit;
+
+import pl.allegro.tech.hermes.management.domain.dc.DatacenterBoundRepositoryHolder;
+import pl.allegro.tech.hermes.management.domain.dc.RepositoryCommand;
+
+class DeleteOfflineRetransmissionTaskCommand extends RepositoryCommand<OfflineRetransmissionRepository> {
```

Comment:
Is there any cleaner way of implementing this commands? I mean for example to create one class `OfflineRetransmissionTaskCommand` instead both `CreateOfflineRetransmissionTaskCommand` and `DeleteOfflineRetransmissionTaskCommand`

Reply:
I think it would be not so easy, because each command has to implement specific interface. If one class would implement operations for both commands, it would be ambiguous for which operation we are actually doing stuff. For example: how to differentiate `execute` method for create/delete command if they are implemented in the same class ?